BigQuery/src/BigQueryClient.php

```diff
@@ -77,32 +77,32 @@ class BigQueryClient
      * @param array $config [optional] {
      *     Configuration options.
      *
-     *     @type string $projectId The project ID from the Google Developer's
+     * @type string $projectId The project ID from the Google Developer's
```

Comment:
I think we need to keep the indentation.

Reply:
Oops my ide did something wrong automatically, fixed